data/remnant/remnant missions.txt

```diff
@@ -3463,3 +3463,268 @@ event "remnant: pantica void sprites"
+			label dawndetails
+			apply
+				"remnant symbionts info" += 1
+			`	"Basically, ages ago we domesticated some of the creatures found here, and discovered that they could be attached to us to form a symbiotic relationship." She pauses again to eat. "I only recently received mine, so I am not fully adjusted to it yet."`
+			choice
+				`	"Okay, that's as much as I need to know."`
+					goto onwards
+				`	"What is it like to have a symbiont?"`
+			`	"Oh, it is wonderful. It is very hard to describe. At first it is just slimy, then a lot of tingling. The tingling lasted for several hours, and then it was just like waking up: rising up out of darkness to a brighter world with just so much to see and learn. Suddenly, it was as if everything was moving slower and I could take a long hard look at everything..." Dawn is visibly excited, signing so fast you have trouble following.`
+			choice
+				`	"Thanks for the description. I should go prep the <ship> now."`
+					accept
+				`	"Where is your symbiont? Yours does not seem to be obvious."`
+					goto dawnwhere
+				`	"Could I see your symbiont?"`
+			`	Dawn looks at you with an amused expression. "We don't know each other nearly well enough for that. Better to focus on the expedition. I will catch up to you on the ship."`
+				accept
+			label onwards
+			`	She nods. "Well, I will finish my meal and meet you on the <ship> shortly." With that, she returns to her meal.`
+				accept
+			label dawnwhere
+			`	Dawn looks at you with an amused expression. "Just as every human is a unique individual, so every symbiont is also unique. As such, they attach to different areas of the body. Mine is located primarily along my spine." She twists around, and you can see a small layer of the tell-tale scaly leather extending up the back of her neck.`
+			`	"We should probably be focusing on the expedition, though. You go get the <ship> ready for launch, and I will be there shortly."`
+				accept
+	on accept
+		event "remnant: lab availability"
+	on complete
+		outfit "Research Laboratory"
+		conversation
+			`You follow the directions of the starport control and settle onto a pad next to the outfitter. You barely have a chance to finish your landing checks before a team of Remnant have a compact pod waiting on your loading ramp. Once you open the cargo bay doors they pick it up and maneuver it inside. They start securing it to the ship while one approaches you and Dawn.`
+			`	"We have loaded the research laboratory onboard. The team will install it if they can find the space, but if they do not, it will be secured in the cargo bay." The Remnant turns to you as she continues. "Before you depart, please verify that the laboratory is installed in the correct location. You know your ship better that we do, after all." She turns back to Dawn and makes a gesture of respect. "Embers guide your path, Seeker." With that, she strides away into the hive of activity that is the outfitting bays.`
```

Comment:
```suggestion
			`	"We have loaded the research laboratory onboard. The team will install it if they can find the space, but if they do not, it will be secured in the cargo bay," The Remnant scientist explains. "Before you depart, please verify that the laboratory is installed in the correct location. You know your ship better that we do, after all." She turns back to Dawn and makes a gesture of respect. "Embers guide your path, Seeker." With that, she strides away into the hive of activity that is the outfitting bays.`
```
Edit for consistency with the suggestion above this one.